core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -146,7 +148,7 @@ public class DefaultBlockWorker extends AbstractWorker implements BlockWorker {
     this(new BlockMasterClientPool(),
         new FileSystemMasterClient(MasterClientContext
             .newBuilder(ClientContext.create(ServerConfiguration.global())).build()),
-        new Sessions(), new TieredBlockStore(), ufsManager);
+        new Sessions(), LocalBlockStore.create(ufsManager), ufsManager);
```

Comment:
I'm assuming that later this will be configurable.

Reply:
done, made the instance type of LocalBlockStore configurable